test/e2e/runner/wait.go

```diff
@@ -30,5 +30,5 @@ func WaitUntil(testnet *e2e.Testnet, height int64) error {
 // waitingTime estimates how long it should take for a node to reach the height.
 // More nodes in a network implies we may expect a slower network and may have to wait longer.
 func waitingTime(nodes int) time.Duration {
-	return time.Duration(20+(nodes*4)) * time.Second
+	return time.Minute + (time.Duration(nodes) * (10 * time.Second))
```

Comment:
Just for comparison, in `v0.34.0` this was hardcoded to 20 seconds flat. Oh how far we've come

Reply:
In practice this doesn't hit as often/far as you'd think, but in some slower tests it's nice to not have to hit it.